phpBB/phpbb/avatar/driver/upload.php

```diff
@@ -281,9 +281,13 @@ public function delete($row)
 		);
 		extract($this->dispatcher->trigger_event('core.avatar_driver_upload_delete_before', compact($vars)));
 
-		if (!sizeof($error) && file_exists($filename))
+		if (!sizeof($error) && $this->filesystem->exists($filename))
 		{
-			@unlink($filename);
+			try
+			{
+				$this->filesystem->remove($filename);
+			}
+			catch(\phpbb\filesystem\exception\filesystem_exception $e) {}
```

Comment:
Yes, I know, I wasn't sure in that part about using \phpbb\filesystem or \phpbb\files